index.js

```diff
@@ -319,7 +336,15 @@ class SafeView extends Component {
         return statusBarHeight(isLandscape);
       }
       case 'bottom': {
-        return isIPhoneX ? (isLandscape ? 24 : 34) : 0;
+        if (isIPhoneX) {
+          return isLandscape ? 24 : 34;
+        }
+
+        if (isNewIPadPro) {
+          return 20;
```

Comment:
Why `20` for iPadPro and not `34` to be the same as the iPhoneX? They both have the "Home Line" and need the same amount of clearance right? @cesargdm